target/linux/ath79/mikrotik/base-files/etc/board.d/01_leds

```diff
@@ -9,6 +9,8 @@ board=$(board_name)
 case "$board" in
 mikrotik,routerboard-lhg-2nd)
 	ucidef_set_led_netdev "lan" "lan" "green:lan" "eth0"
+mikrotik,routerboard-mapl-2nd)
+	ucidef_set_led_netdev "lan" "lan" "green:eth" "eth0"
```

Comment:
Missing ";;". Thanks for showing us that this is not tested.

Reply:
You are welcome :(
Sorry, I have only one device and it's in use (running OpenWrt), so I can not run-test every change...

Reply:
Well, would have saved you from posting a non-running version.